packages/commands-http/src/request.ts

```diff
@@ -0,0 +1,84 @@
+import { Command } from '@seagull/commands'
+import fetch, { Response as FetchResponse } from 'node-fetch'
+import { Http, RequestConfig } from './index'
+import SeedStorage from './seedStorage'
+
+/**
+ * Http request command.
+ */
+export class Request<T> extends Command<T> {
+  executeConnected = this.executeCloud
+  executeEdge = this.executeCloud
+
+  private seed: SeedStorage<T>
+
+  /**
+   * Creates an http request command.
+   * @param config Http request command configuration.
+   */
+  constructor(public config: RequestConfig) {
+    super()
+    if (!config) {
+      throw new Error('HttpCommand: You have to specify a config!')
+    }
+    if (!this.config.url) {
+      throw new Error('HttpCommand: The url in the config is not specified!')
+    }
+    this.config = config
+    this.seed = SeedStorage.createByRequest(config.url, config.init)
+  }
+
+  async executeCloud(): Promise<T> {
+    const response = await fetch(this.config.url, this.config.init)
+    switch (this.config.parseBody) {
+      case 'xml':
+        throw new Error('not implemented')
+      case 'text':
+        return (await response.text()) as any
```

Comment:
Using `response.text()` does not convert the data into UTF8, so there could potentially be encoding errors.

Reply:
Guess we could argument on this both ways. For now i don't now any case where this would be needed. Does anyone have an opinion on always converting text to utf-8? 

Note: For now only Maik is using this for parsing XML response data strings.

Reply:
Right now, iris parses XML data with a separate parser, which is addressed with the 'xml' parseBody configuration. The 'text' option is used for parsing crm data, haven't checked this, as it's not my implementation, doing it right now :)
Note: In case we want to encode responses to utf-8, we might want to to this before seed generation, because right now we don't save header data.

Reply:
No headers, CRM XML encoding is not defined, so no idea how to check or change the encoding. Do you have an idea? BOM?

Reply:
Ah, now i see, you prefer to use _body.textConverted()_ instead of _body.text()_. How about creating a separate parseBody mode like 'textUtf8'? Not sure if that would do anything in this case though...

Reply:
implemented textConverted as default, let's try if that produces any errors in the future ...